### ISSUE TITLE
Player process 1.0

### ISSUE THREAD

**Issue by justheuristic** [yandexdataschool/tinyverse#3]
A process that
- compiles the agent
- takes the stored dqn params every once in a while
 - if there are no params, put them there
- interacts :) (cpu)
- stores the interactions in the database
- all interactions are of same length (e.g. 10 ticks)
- loads new weights every once in a while
- does not break down everything if restarted at random point of time
- does not erase old sessions
- does not intentionally break down if there are several such processes running in parallel :)

Basically do [that](https://github.com/yandexdataschool/tinyverse/blob/master/tests/test_database.py) but with an agent and all the trouble that comes with it. 
[Example agent setup](https://github.com/justheuristic/vime/blob/master/mountaincar-qlearning.ipynb) for mountaincar.
